### PR TITLE
Configuring for pure-python

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by gforcada** [comment]
@icemac I'm no admin in the `zopefoundation` organization nor on this repository, so I can not remove the `py27` check that is marked as required, would you mind doing that? 🍀 🙇🏾

**Comment by dataflake** [comment]
Did you create the PR with the script `drop-legacy-python.py` in meta/config? That also adds a change log entry and guides you what else to look for.

**Comment by icemac** [comment]
> Did you create the PR with the script `drop-legacy-python.py` in meta/config? That also adds a change log entry and guides you what else to look for.

Additionally this script asks you to remove `six` and runs `pyupgrade`, so I'd prefer a PR using this script.

**Comment by gforcada** [comment]
I was not aware of that script, let me put this one on draft then and use that instead. Probably tomorrow 😴

**Comment by gforcada** [comment]
I did run `drop-legacy-python.py` and fixed the `tox -e lint` errors.

Let me know if you want more changes to be made 😄

File: setup.py
Comment (dataflake) [inline setup.py:1]: Please change `author_email` to zope-dev@zope.dev and add `python_requires='>=3.7'`. Those are non-automated changes that we always do manually where needed.

**Review by dataflake — CHANGES_REQUESTED** [review]
Just a few small items that are not automated

File: setup.py
Comment (gforcada) [reply setup.py:1]: Thanks, did that and dropped `six` from `install_requires` as well 😄

**Review by dataflake — APPROVED** [review]

**Comment by dataflake** [comment]
You don't need to manually re-request a review, I get email for all changes as an active reviewer, anyway, and will look at them when I have a moment.

**Pull request merged**